articles/introduction-to-deep-learning/index.md

```diff
@@ -0,0 +1,123 @@
+1. Self-driven cars
+
+Companies such as Google are building driver assistance services. Also, they are teaching computers how to use digital sensors instead of human sensors. These companies train Deep Learning algorithms to use [large amounts of data](https://medium.com/breathe-publication/top-15-deep-learning-applications-that-will-rule-the-world-in-2018-and-beyond-7c6130c43b01).
+
+2. Natural Language Processing
+
+Machines are taught to understand complexities associated with languages such as semantics. To achieve this, [NLP through Deep Learning](https://www.mygreatlearning.com/blog/natural-language-processing-tutorial/) plays a significant role. NLP also catch linguistic nuances and frame appropriate responses.
+
+3. Healthcare
+
+Deep Learning is completely revolutionizing the healthcare and the medical industries. According to [Medium](https://medium.com/breathe-publication/top-15-deep-learning-applications-that-will-rule-the-world-in-2018-and-beyond-7c6130c43b01), AI has enabled healthcare and medical industries to advance tremendously.
+Clinical researchers use DL to find a cure for untreatable diseases. DL helps with a speedy diagnostic of dangerous conditions. Many cancer tests, such as the Pap test and Mammograms, use DL to examine cells image using a microscope.
+
+4. Virtual Assistants
+
+Siri and Google Assistants are approved deep learning virtual assistants. Deep Learning enables virtual assistants to learn and understand commands given by a user. Virtual assistants then execute by providing the appropriate answer naturally. Virtual assistants use Deep Learning to learn about you, from what you search most.
+
+5. Fraud Detection
+
+Banking and financial sectors are benefiting from Deep Learning to detect transaction fraud. [Autoencoders](https://www.mygreatlearning.com/blog/deep-learning-applications/) in Tensorflow are being used to catch credit card fraud, thus saving a lot of money from fraudsters. Fraud prevention is done based on recognizing patterns in customer transactions.
+
+6. Image Recognition
+
+Image Recognition using Deep Learning aims to recognize and learn content in images. Also, Deep Learning aims to understand their surrounding. Image Recognition is mostly used in gaming and social media.
+
+7. Entertainment
+
+Entertainment industries such as [Netflix](https://www.netflix.com/ke-en/) recommends to their viewers what they need. Deep Learning enables the entertainment industry to understand the consumer's behavior. This helps to provide an exciting experience to clients. Deep Learning is revolutionizing the filmmaking process. Cameras can learn body language and conduct voice synthesis. Deep Learning also helps to emulate someone's voice to imbibe in virtual characters.
+
+### Working of Neural Networks
+Neural Networks are algorithms modeled to function like the human brain. Neural Networks is a system of hardware and software patterned after neurons operation in the human mind. Neurons are decision-makers.
```

Comment:
They are not modeled to function as the brain. They are simply inspired by the brain. 

Neurons are not the decision-makers. They are computational units. The decision is taken by the loss function and the optimization process.